src/bin/ion/commands/beta/inspect.rs

```diff
@@ -509,9 +514,7 @@ impl<'a> IonInspector<'a> {
             to_hex(&mut self.hex_buffer, self.reader.raw_value_bytes().unwrap());
         }
 
-        const TYPE_DESCRIPTOR_SIZE: usize = 1;
-        let length =
-            TYPE_DESCRIPTOR_SIZE + self.reader.header_length() + self.reader.value_length();
+        let length = self.reader.header_length() + self.reader.value_length();
```

Comment:
`ion-rs` v0.12.0 changed the `header_length()` method to include the type descriptor byte, so we no longer have to account for it manually.